lib/transforms/analyzer.cc

```diff
@@ -55,8 +55,7 @@ Analyzer::NodeInfo& GenerateCommonInfo(
   node_info.name = inst->GetName();
   node_info.type = inst->GetOpCode();
   node_info.data_type = inst->GetOperand(0).GetType().GetDataType();
-  // node_info.input_shape =
-  // inst->GetOperand(0).GetType().GetDimSizes();
+  node_info.input_shape = inst->GetOperand(0).GetType().GetDimSizes();
   node_info.output_shape = inst->GetResultType().GetDimSizes();
```

Comment:
in case of multiple inputs/outputs, we'd better to make input_shape to `std::vector<std::vector<int64_t>>`

Reply:
Agreed, can you or Hongyao to follow up this issue? I just want to see input shape of resent and mobilenet with the line uncommented.

Reply:
OK